src/js/parsers/mobiledoc.js

```diff
@@ -46,15 +50,18 @@ export default class MobiledocParser {
   parseSection(section, post) {
     let [type] = section;
     switch(type) {
-      case 1: // markup section
+      case MOBILEDOC_MARKUP_SECTION_TYPE:
         this.parseMarkupSection(section, post);
         break;
-      case IMAGE_SECTION_TYPE:
+      case MOBILEDOC_IMAGE_SECTION_TYPE:
         this.parseImageSection(section, post);
         break;
-      case CARD_SECTION_TYPE:
+      case MOBILEDOC_CARD_SECTION_TYPE:
         this.parseCardSection(section, post);
         break;
+      case MOBILEDOC_LIST_SECTION_TYPE:
```

Comment:
un-certain about leaking `MOBILEDOC` onto the type names. There is nothing in the post abstract that is mobiledoc specific. `POST_` or `ABSTRACT_` or `AST_`?